examples/gcp/gke-existing_cluster/variables.tf

```diff
@@ -51,20 +48,30 @@ variable "existing_vpc_name" {
   type        = string
 }
 
-variable "existing_subnet_name" {
-  description = "The name of the existing Subnet"
+variable "existing_vpc_id" {
```

Comment:
We don't need both `existing_vpc_name` and `existing_vpc_id`. Can use one to query the other.